client/src/app/shared/components/choice-dialog/choice-dialog.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+
+interface ChoiceDialogData {
+    title: string;
+    content: string;
+    choices: Object;
```

Comment:
`Object` or `object`? I suppose it would be object.
Object brings the full inheritance while "object" is a primary datatype